apps/web/public/static/locales/en/common.json

```
@@ -3590,6 +3594,7 @@
   "pbac_desc_create_availability": "Create availability",
   "pbac_desc_view_availability": "View availability",
   "pbac_desc_update_availability": "Update availability",
+  "add_people_to_blocklist": "Add people emails or domains to your blocklist to prevent them from booking your event types",
```

```suggestion
  "add_people_to_blocklist": "Add email addresses or domains to your blocklist to prevent them from booking your event types",
```

I think that's better, I know the designs show it as you have it